docs/workflow/building/libraries/README.md

```diff
@@ -184,6 +184,13 @@ dotnet build System.Net.NetworkInformation.csproj /p:OSGroup=Linux
 dotnet build -c Release System.Net.NetworkInformation.csproj
 ```
 
+### Building for Mono
+By default the libraries will attempt to build using the CoreCLR version of `System.Private.CoreLib.dll`. In order to build against the Mono version you need to use the `/p:RuntimeFlavor=Mono` argument.
+
+```
+.\build.cmd -subsetCategory libraries /p:RuntimeFlavor=Mono
```

Comment:
```suggestion
./build.sh -subsetCategory libraries /p:RuntimeFlavor=Mono
```

for consistency with the rest of the doc which mostly uses build.sh

Reply:
I specifically chose windows here because I haven't actually validated Linux yet. I was going to use Linux Training day (aka today) to take this change and validate on Linux. As a part of doing that I was planning on updating the doc.